modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/SearchHistoryDao.kt

```diff
@@ -0,0 +1,23 @@
+package au.com.shiftyjelly.pocketcasts.models.db.dao
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import au.com.shiftyjelly.pocketcasts.models.entity.SearchHistoryItem
+
+@Dao
+abstract class SearchHistoryDao {
+    @Query("SELECT * FROM search_history ORDER BY modified DESC LIMIT :limit")
+    abstract fun findAll(limit: Int = 10): List<SearchHistoryItem>
```

Comment:
Did you intend to not make this a suspended function? If you want to make that change, feel free to address that in one of your follow-up PRs when you start using this DAO.

Reply:
Added in 24ea15c, thanks for pointing it out!